<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work bench</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QTableView" name="tbvInvoice"/>
      </item>
      <item row="1" column="1">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbKey">
          <property name="text">
           <string>Ключ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leKey">
          <property name="text">
           <string>лон</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="5">
           <widget class="QLineEdit" name="leVal6"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="leVal1"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leVal2"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="lbPatt5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="lbPatt4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="leVal4"/>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="leVal5"/>
          </item>
          <item row="1" column="6">
           <widget class="QLineEdit" name="leVal7"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="leVal3"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbPatt1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lbPatt3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbPatt2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="lbPatt7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="lbPatt6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="mFileExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Редактор</string>
    </property>
    <addaction name="mKeyEdit"/>
    <addaction name="mPattEdit_2"/>
    <addaction name="mParamEdit"/>
    <addaction name="mProdEdit"/>
    <addaction name="mManEdit"/>
    <addaction name="mMultEdit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Импорт данных</string>
    </property>
    <addaction name="mNomenclEdit"/>
    <addaction name="mPriceEdit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Разработка</string>
    </property>
    <addaction name="mClearProducts"/>
    <addaction name="mClearkeywords"/>
    <addaction name="mClearManufacturers"/>
    <addaction name="mClearMultiplicy"/>
    <addaction name="mClearParams"/>
    <addaction name="mClearPatterns"/>
    <addaction name="mClearStore"/>
    <addaction name="mClearStore_date"/>
    <addaction name="mClearStoreMan"/>
    <addaction name="mClearAll"/>
   </widget>
   <addaction name="mFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mFileExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="mPattEdit">
   <property name="text">
    <string>Редактор</string>
   </property>
  </action>
  <action name="mKeyEdit">
   <property name="text">
    <string>Ключи</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="mPattEdit_2">
   <property name="text">
    <string>Шаблоны</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="mParamEdit">
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
  <action name="mProdEdit">
   <property name="text">
    <string>Продукты</string>
   </property>
  </action>
  <action name="mManEdit">
   <property name="text">
    <string>Производители</string>
   </property>
  </action>
  <action name="mNomenclEdit">
   <property name="text">
    <string>Номенклатура</string>
   </property>
  </action>
  <action name="mPriceEdit">
   <property name="text">
    <string>Прайс</string>
   </property>
  </action>
  <action name="mMultEdit">
   <property name="text">
    <string>Кратность упаковки</string>
   </property>
  </action>
  <action name="mClearProducts">
   <property name="text">
    <string>Очистить &quot;products&quot;</string>
   </property>
  </action>
  <action name="mClearkeywords">
   <property name="text">
    <string>Очистить &quot;keywords&quot;</string>
   </property>
  </action>
  <action name="mClearManufacturers">
   <property name="text">
    <string>Очистить &quot;manufacturers&quot;</string>
   </property>
  </action>
  <action name="mClearMultiplicy">
   <property name="text">
    <string>Очистить &quot;multiplicy&quot;</string>
   </property>
  </action>
  <action name="mClearParams">
   <property name="text">
    <string>Очистить &quot;params&quot;</string>
   </property>
  </action>
  <action name="mClearPatterns">
   <property name="text">
    <string>Очистить &quot;patterns&quot;</string>
   </property>
  </action>
  <action name="mClearStore">
   <property name="text">
    <string>Очистить &quot;store&quot;</string>
   </property>
  </action>
  <action name="mClearStore_date">
   <property name="text">
    <string>Очистить &quot;store_date&quot;</string>
   </property>
  </action>
  <action name="mClearStoreMan">
   <property name="text">
    <string>Очистить &quot;store_manufacturer&quot;</string>
   </property>
  </action>
  <action name="mClearAll">
   <property name="text">
    <string>Очистить все таблицы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leKey</tabstop>
  <tabstop>leVal1</tabstop>
  <tabstop>leVal2</tabstop>
  <tabstop>leVal3</tabstop>
  <tabstop>leVal4</tabstop>
  <tabstop>leVal5</tabstop>
  <tabstop>leVal6</tabstop>
  <tabstop>leVal7</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mFileExit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mKeyEdit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMKeyEditTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPattEdit_2</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMPattEditTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mParamEdit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMParamEditTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mProdEdit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMProdEditTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leVal1</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotValLeEdtFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leVal3</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotValLeEdtFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leVal4</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotValLeEdtFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leVal6</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotValLeEdtFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leVal7</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotValLeEdtFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leVal2</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotValLeEdtFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leVal5</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotValLeEdtFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mManEdit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMManEditTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mNomenclEdit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMNomEditTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPriceEdit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotMPriceEditTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>testslot()</slot>
  <slot>leKeyEdFinished()</slot>
  <slot>mPattEditTrigg()</slot>
  <slot>slotMKeyEditTriggered()</slot>
  <slot>slotMPattEditTriggered()</slot>
  <slot>slotMParamEditTriggered()</slot>
  <slot>slotMProdEditTriggered()</slot>
  <slot>slotValLeEdtFinished()</slot>
  <slot>slotMManEditTriggered()</slot>
  <slot>slotMNomEditTriggered()</slot>
  <slot>slotMPriceEditTriggered()</slot>
 </slots>
</ui>
